onnx2torch/node_converters/slice.py

```diff
@@ -119,7 +125,17 @@ def _(node: OnnxNode, graph: OnnxGraph) -> OperationConverterResult:  # pylint:
 @add_converter(operation_type='Slice', version=11)
 @add_converter(operation_type='Slice', version=13)
 def _(node: OnnxNode, graph: OnnxGraph) -> OperationConverterResult:  # pylint: disable=unused-argument
+    constant_params = []
+    params_names = node.input_values[1:]
+    for name in params_names:
```

Comment:
What do you think about using a ilst comprehension as below? This should result in a single statement being translated by python's compiler, resulting in increased efficiency.

```constant_params = [get_const_value(name, graph).tolist() if name in graph.initializers or name in graph._node_output_values else [] for name in params_names]```